Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2734 @@
+    """
+    Check the command status of isolate endpoint.
+    Args:
+        client: client object to use.
+        args: all command arguments, usually passed from ``demisto.args()``.
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains an updated
+            result.
+    """
+    command_id = args.get('command_id', '')
+    readable_data = []
+    payload = post_request_body(args)
+    payload.pop('offset', 0)
+
+    raw_response = client.get_status_endpoint(command_id, payload)
+
+    summary_data = {
+        "state": raw_response.get('state'),
+        "Command Issuer Name": raw_response.get('command_issuer_name'),
+    }
+
+    if result := raw_response.get('status', ()):
+        for status in result:
+            summary_data['state'] = status.get('state', '')
+            summary_data['message'] = status.get('message', '')
+            summary_data['error_code'] = status.get('error_code', '')
+
+    if summary_data:
+        title = "Command Status"
+        readable_data.append(summary_data)
+        readable_output = generic_readable_output(readable_data, title)
+    else:
+        readable_output = 'No command status data to present.'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.CommandStatus',
+        outputs_key_field='',
+        outputs=summary_data,
+        raw_response=raw_response,
+        ignore_auto_extract=True
+    )
+
+
+''' FETCHES INCIDENTS '''
+
+
+def fetch_incidents(client: Client) -> list:
+    """
+    Arguments Validation.
+    Args:
+        client: Client Object
+    Returns:
+        Incidents List
+    """
+
+    rev_incident_priority = {v: k for k, v in INCIDENT_PRIORITY_LEVEL.items()}
+    rev_incident_state = {v: k for k, v in INCIDENT_STATUS.items()}
+
+    seperator = ' OR '
+    priority_list = [rev_incident_priority.get(i) for i in client.fetch_priority]  # type: ignore[union-attr]
+    priority = priority_list[0] if len(priority_list) == 1 else seperator.join(map(str, priority_list))
+
+    state_list = [rev_incident_state.get(i) for i in client.fetch_status]  # type: ignore[union-attr]
+    state = state_list[0] if len(state_list) == 1 else seperator.join(map(str, state_list))
+
+    last_run = demisto.getLastRun()
+    demisto.debug(f'Last Run Object : {last_run}')
+
+    payload = {
+        "verb": "query",
+        "limit": client.fetch_limit,
+        "query": f'priority_level: ({priority}) AND state: ({state})'
+    }
+    demisto.debug(f'limit: {client.fetch_limit}, priority_level: ({priority}) AND state: ({state})')
+    # demisto.getLastRun() will return an obj with the previous run in it.
+    # set First Fetch starting time in case running first time or reset
+    start_time = convert_to_iso8601(client.first_fetch)
+    start_time_n, end_time = get_fetch_run_time_range(last_run=last_run, first_fetch=client.first_fetch)
```

Comment:
what does `_n` represent?

Reply:
Refactoring the fetch_incident code kindly review.